code/modules/library/lib_items.dm

```diff
@@ -61,7 +61,7 @@
 				state = 0
 
 		if(2)
-			if(istype(I, /obj/item/weapon/book) || istype(I, /obj/item/weapon/spellbook))
+			if(istype(I, /obj/item/weapon/book) || istype(I, /obj/item/weapon/spellbook) || istype(I, /obj/item/weapon/storage/book))

```

Comment:
The bible is a subtype of the hollowed book, so it'll work by default.
I'll make the other change with the lists though.